src/Test/Components.test.jsx

```diff
@@ -6,53 +6,38 @@ import { render } from '@testing-library/react';
 import { Provider } from 'react-redux';
 import store from '../redux/store';
 import MainPage from '../components/MainPage';
-import CountriesItem from '../components/CountiresItem';
+import CountiresItem from '../components/CountiresItem';
 import CountryDetails from '../components/CountryDetails';
 import DetailsPage from '../components/DetailsPage';
 
-jest.mock('../components/MainPage');
-jest.mock('../components/CountiresItem');
-jest.mock('../components/CountryDetails');
-jest.mock('../components/DetailsPage');
-
-describe('Testing components', () => {
-  it('renders MainPage component', () => {
-    const mainPage = render(
-      <Provider store={store}>
-        <MainPage />
-      </Provider>,
-    );
-    expect(mainPage.getByTestId("main-page")).toBeInTheDocument();
-    expect(mainPage).toMatchSnapshot();
-  });
+test('renders MainPage component', () => {
+  render(
+    <Provider store={store}>
+      <MainPage />
+    </Provider>,
+  );
```

Comment:
- [ ]  Great job implementing the required changes I requested you thank you so much but there are still some issues you need to address before this project can be approved . According to the requirement, It's possible to navigate the detail page by clicking the individual category item, there's a link from the Detail page to the Home page by clicking the "Back" button. please fix this 
- [ ] given desine
<img width="421" alt="metrics_app" src="https://github.com/CarlosZ96/countries-database/assets/92458236/3e1b01bd-00ea-4ef7-bdd0-a0dcd4bc3d85">

- [ ] your design
![Screenshot 2023-09-24 032435](https://github.com/CarlosZ96/countries-database/assets/92458236/a3b8c81a-0dcd-40ae-bfae-1e084b5c1392)